doc/sphinx/src/examples.rst

```diff
@@ -0,0 +1,65 @@
+.. _examples:
+
+Examples
+=========
+
+The ``examples`` directory of ``singularity-eos`` contains several
```

Comment:
```suggestion
The ``example`` directory of ``singularity-eos`` contains several
```

The directory is `example` so just a minor change for consistency or if you want to rename it to examples that works as well.

Reply:
good catch. Thanks!